.github/actions/fetch-codeql/action.yml

```
@@ -8,7 +8,7 @@ runs:
       run: |
         LATEST=$(gh release list --repo https://github.com/github/codeql-cli-binaries | cut -f 1 | grep -v beta | sort --version-sort | tail -1)
         gh release download --repo https://github.com/github/codeql-cli-binaries --pattern codeql-linux64.zip "$LATEST"
-        unzip -q codeql-linux64.zip
-        echo "${{ github.workspace }}/codeql" >> $GITHUB_PATH
+        unzip -q -d "${{ runner.temp }}" codeql-linux64.zip
+        echo "${{ runner.temp  }}/codeql" >> $GITHUB_PATH
```

Minor. This is safe as written, but I generally prefer to use the associated environment variables rather than inlining `${{ }}` contexts since not all contexts are escaped.

Out of curiosity, why temp and not the workspace?
```suggestion
        unzip -q -d "$RUNNER_TEMP" codeql-linux64.zip
        echo "$RUNNER_TEMP/codeql" >> "$GITHUB_PATH"
```

---

Yes that is a good change, thanks! I changed to using the temp folder to avoid collisions with files that may be in the workspace.